kratos/tests/cpp_tests/geometries/test_triangle_3d_6.cpp

```diff
@@ -61,7 +61,7 @@ namespace {
 
     KRATOS_TEST_CASE_IN_SUITE(Triangle3D6Volume, KratosCoreGeometriesFastSuite) {
         auto geom = GenerateReferenceTriangle3D6();
-        KRATOS_CHECK_EXCEPTION_IS_THROWN(geom->Volume(), "Calling base class 'Volume' method instead of derived class one.");
```

Comment:
This one for instance.

Reply:
I'm not discussing if this whether makes sense or not, my point is that, in my opinion, it is worth discussing these again as the PR is quite old and these are crucial classes. @KratosMultiphysics/technical-committee

Reply:
Understandable, just explaining my point